### DISCUSSION TITLE
DVC pull from Amazon S3

### DISCUSSION THREAD

**Discussion by Alberto1404** [treeverse/dvc#9081] [Help]
Hello everyone. I would like to use dvc for big data versioning. I have two gitlab repositories, local and global. Global is a mirror of the local, such that we only perform changes in the local one, and then clone the global repository in the computers to use those big data files. (let's call them destiny machines, **which are not connected to our LAN**)
The idea is to use a AWS s3 bucket to store the data. I have successfully created the bucket and pushed my files there via ```dvc push```
My question is how to prepare the destiny machine in order to be able to perform ```dvc pull``` from that s3 bucket. 
Apart from installing dvc, should I do as well the ```dvc remote add -d mydvc s3://path/to/my/bucket```?
Is there any other relevant step am I missing?
Thank you

**Comment by daavoo** [comment]
Hi @Alberto1404 , `dvc remote add -d` will just write to `.dvc/config`. You can track that file with git in order to be present in the repository, see https://dvc.org/doc/command-reference/config#description
